apps/webbsite/next.config.js

```diff
@@ -12,6 +12,9 @@ const nextConfig = {
     // See: https://github.com/gregberge/svgr
     svgr: true,
   },
+  images: {
+    unoptimized: true,
+  },
```

Comment:
Why do we want unoptimized images here? @dutterbutter

Reply:
@AtelyPham good eye. It is a requirement for deploying static Next.js websites. You can view the unsupported features for static exports here in their docs. https://nextjs.org/docs/advanced-features/static-html-export#unsupported-features

Reply:
@AtelyPham we can make use of a custom loader to optimize these images just not the default one as mentioned in the supported features part of the docs: https://nextjs.org/docs/advanced-features/static-html-export#supported-features